### PR TITLE
Explain how to resolve ambiguous type

### PR DESCRIPTION
**Pull request by danielmarbach**

### CODE REVIEW

File: nservicebus/upgrades/host-6to7.md [hunk 1]
```diff
@@ -47,4 +47,17 @@ WARNING: If an `EndpointConfig.cs` file already exists in the project, be carefu
...
 
-The WCF integration has been augmented with additional functionality such as the ability to reply with messages to the client, cancel requests and reroute to other endpoints. More information can be found in [Wcf](/nservicebus/wcf).
\ No newline at end of file
+The WCF integration has been augmented with additional functionality such as the ability to reply with messages to the client, cancel requests and reroute to other endpoints. More information can be found in [Wcf](/nservicebus/wcf).
+
+### Ambiguous type compilation error
+
+By default referenced assemblies are imported into the global namespace. This might lead to ambiguous type reference problems when implementing `WcfService<TRequest, TResponse>` or `IWcfService<TRequest, TResponse>`. To resolve the ambiguous type reference do the following:
```
Comment (dvdstelt) [inline nservicebus/upgrades/host-6to7.md]: Should we add a link for global namespace, as in [global namespace](https://msdn.microsoft.com/en-us/library/z2kcy19k.aspx#Anchor_0)? People might think it's something else.

File: nservicebus/upgrades/host-6to7.md [hunk 2]
```diff
@@ -47,4 +47,17 @@ WARNING: If an `EndpointConfig.cs` file already exists in the project, be carefu
...
+
+- In the Solution Explorer right click under References on the `NServiceBus.Wcf` reference and switch to properties, or use `Alt+Enter`
+- Under the alias property set an alias for the assembly, i.ex. `wcf`
```
Comment (dvdstelt) [inline nservicebus/upgrades/host-6to7.md]: What does "i.ex." mean?

File: nservicebus/upgrades/host-6to7.md [hunk 3]
```diff
@@ -47,4 +47,17 @@ WARNING: If an `EndpointConfig.cs` file already exists in the project, be carefu
...
 
-The WCF integration has been augmented with additional functionality such as the ability to reply with messages to the client, cancel requests and reroute to other endpoints. More information can be found in [Wcf](/nservicebus/wcf).
\ No newline at end of file
+The WCF integration has been augmented with additional functionality such as the ability to reply with messages to the client, cancel requests and reroute to other endpoints. More information can be found in [Wcf](/nservicebus/wcf).
+
+### Ambiguous type compilation error
+
+By default referenced assemblies are imported into the global namespace. This might lead to ambiguous type reference problems when implementing `WcfService<TRequest, TResponse>` or `IWcfService<TRequest, TResponse>`. To resolve the ambiguous type reference do the following:
+
+- In the Solution Explorer right click under References on the `NServiceBus.Wcf` reference and switch to properties, or use `Alt+Enter`
```
Comment (dvdstelt) [inline nservicebus/upgrades/host-6to7.md]: I'd call it "properties pane", that's more clear and distincts it from the properties window of a project.

**Comment by danielmarbach** [comment]
@dvdstelt addressed your feedback

**Pull request merged**